src/main/java/com/spotify/reaper/resources/RepairRunResource.java

```diff
@@ -438,34 +438,51 @@ private URI buildRepairRunURI(UriInfo uriInfo, RepairRun repairRun) {
    */
   @GET
   public Response listRepairRuns(@QueryParam("state") Optional<String> state) {
-    Set desiredStates = splitStateParam(state);
-    if (desiredStates == null) {
-      return Response.status(Response.Status.BAD_REQUEST).build();
+    try {
+      List<RepairRunStatus> runStatuses = Lists.newArrayList();
+      Set desiredStates = splitStateParam(state);
+      if (desiredStates == null) {
+        return Response.status(Response.Status.BAD_REQUEST).build();
+      }
+      Collection<RepairRun> runs;
+      
+      Collection<Cluster> clusters = context.storage.getClusters();
+      for (Cluster cluster : clusters) {
+        runs = context.storage.getRepairRunsForCluster(cluster.getName());
+        runStatuses.addAll(getRunStatuses(runs, desiredStates));
+      }
+      
+      return Response.status(Response.Status.OK).entity(runStatuses).build();
+    } catch (Exception e) {
+      // TODO Auto-generated catch block
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)